GameData/RealismOverhaul/Engine_Configs/RL10_Config.cfg

```diff
@@ -7,7 +7,7 @@
 //	RL10A-1
 //	Atlas-Centaur A/B/C
 //
-//	Dry Mass: 145 Kg
+//	Dry Mass: 131 Kg
```

Comment:
Some of these mass changes are fairly significant (10%).  Are our previous sources wrong, or were the values just guess?  I'd prefer not to flip between different sources without something to declare why one is more reliable.

Reply:
I rarely trust net sources myself, NTRS and DTIC.mil however should be very concise.
We have very accurate numbers for an RL-10A-3-3,  That is 289.79lbs with 3 sigma of 290lbs that is to say (99.73%) of all RL10A-3-3 engines will weight between 289.58 and 290.00 lbs (~131kg)
Instrumentation and hydraulic brackets add 10.17lbs (~4.6kg, these should be avionics weights)

https://ntrs.nasa.gov/api/citations/19670005471/downloads/19670005471.pdf